app/views/dashboard/index.html.erb

```diff
@@ -16,8 +16,32 @@
   </thead>
 
   <tbody>
-    <% @reasons.each do |reason| %>
-      <tr class="<%= "active text-muted inactive-reason" if reason.inactive %>">
+    <% @active_reasons.each do |reason| %>
+      <tr class="">
+        <td><%= link_to reason.reason_name, "/reason/#{reason.id}" %></td>
```

Comment:
There's a [route helper](https://github.com/Charcoal-SE/metasmoke/blob/master/config/routes.rb#L128) for this - instead of `"/reason/#{reason.id}"`, you can use `reason_path(reason)`.